api4/webhook.go

```diff
@@ -194,10 +194,16 @@ func getIncomingHook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.LogAudit("fail - bad permissions")
 			c.SetPermissionError(model.PERMISSION_MANAGE_WEBHOOKS)
 			return
-		} else {
-			w.Write([]byte(hook.ToJson()))
+		}
+
+		if c.Session.UserId != hook.UserId && !c.App.SessionHasPermissionToTeam(c.Session, hook.TeamId, model.PERMISSION_MANAGE_OTHERS_WEBHOOKS) {
+			c.LogAudit("fail - inappropriate permissions")
```

Comment:
I think the proper terminology for this is "access denied". Sure the user's *attempt* failed but that's not the convention to name it thus because the system did not fail. What about something like...

`fmt.Sprintf("access denied: missing %v permission", model.PERMISSION_MANAGE_OTHERS_WEBHOOKS.Id)`?

Reply:
I like the idea of have some kind of "uniform error format", actually we can encapsulate in a function, something like `LogAuditAccessDenied(model.PERMISSION_MANAGE_OTHERS_WEBHOOKS)`.

I think this could be in another ticket, because this strings are allover the code.